devito/symbolics/inspection.py

```diff
@@ -110,7 +110,8 @@ def estimate_cost(exprs, estimate=False):
 estimate_values = {
     'elementary': 100,
     'pow': 50,
-    'div': 5
+    'div': 5,
+    'Abs': 5,
```

Comment:
really? Isn't it a single byte change

Reply:
so, before this change the associated cost would have been 100 (elementary) 😬 

if you google, you'll see there's no straightforward answer. Maybe with integers is like you say, but in general I'm not sure. After 60 secs of googling I gave up and decided to put 5. What really matters is that it's not 100 anymore